src/main/webapp/view/discussion/discussion-area.js

```diff
@@ -91,11 +93,21 @@ export default class DiscussionArea {
     this.#currentTimeMs = timeMs;
     DiscussionArea.#ELEMENT_TIMESTAMP_SPAN.innerText =
         timestampToString(timeMs);
-    const nearestComments = this.getNearestDiscussionComments(timeMs);
-    if (nearestComments.length == 0) {
+
+    // Remove any existing highlights.
+    for (const comment of this.#nearestComments) {
+      comment.setHighlighted(false);
+    }
+
+    // Scroll to and highlight new nearest comments.
+    this.#nearestComments = this.getNearestDiscussionComments(timeMs);
+    if (this.#nearestComments.length == 0) {
       return;
     }
-    nearestComments[0].scrollToTopOfDiscussion();
+    this.#nearestComments[0].scrollToTopOfDiscussion();
+    for (const comment of this.#nearestComments) {
+      comment.setHighlighted(true);
+    }
   }
```

Comment:
Same here

Reply:
individual comments don't have access to nearestcomments